src/link/resolvePeers.ts

```diff
@@ -178,7 +177,7 @@ function resolvePeersOfNode (
       hardlinkedLocation,
       independent,
       optionalDependencies: node.pkg.optionalDependencies,
-      children: Object.assign(children, resolvedPeers),
```

Comment:
This one (and the one at line 142) you cannot convert to the spread operator because children and unknownResolvedPeersOfChildren are mutated. They are mutated for performance reasons